tensorboard/webapp/metrics/store/metrics_selectors_test.ts

```diff
@@ -863,29 +863,16 @@ describe('metrics selectors', () => {
     });
   });
 
-  describe('#getPromoteTimeSeries', () => {
-    beforeEach(() => {
-      selectors.getPromoteTimeSeries.release();
-    });
-
-    it('returns current visualization filters', () => {
-      const state = appStateFromMetricsState(
-        buildMetricsState({promoteTimeSeries: false})
-      );
-      expect(selectors.getPromoteTimeSeries(state)).toEqual(false);
-    });
-  });
-
   describe('#isMetricsSettingsPaneOpen', () => {
     beforeEach(() => {
-      selectors.getPromoteTimeSeries.release();
+      selectors.isMetricsSettingsPaneOpen.release();
```

Comment:
So was this test incorrect before?

Reply:
That's right. Looks like a copy & paste error.